light-zk.js/src/test-utils/testChecks.ts

```diff
@@ -17,10 +16,11 @@ import { Program } from "@coral-xyz/anchor";
  * merkle tree is locked by updateState account
  * lock has been taken less than 5 slots ago
  */
-let CONFIRMATION = {
-  preflightCommitment: "confirmed",
-  commitment: "confirmed",
-};
+
+// let CONFIRMATION = {
+//   preflightCommitment: "confirmed",
+//   commitment: "confirmed",
+// };
```

Comment:
assuming it's not used, it can be removed

Reply:
removed ✅